timesheet_helpdesk_ticket/models/ticket_timesheet.py

```diff
@@ -0,0 +1,59 @@
+from odoo import models, fields, api
+
+class TicketTimesheet(models.Model):
+    _inherit = 'helpdesk.ticket'
+
+    partner_phone = fields.Char(
+        string='Contact Phone',
+        related='partner_id.mobile',
+        store=True,
+        readonly=False,
+    )
+    project_id = fields.Many2one(
+        comodel_name='project.project',
+        string='Project',
+    )
+    task_id = fields.Many2one(
+        comodel_name='project.task',
+        string='Task',
+    )
+
+    timesheet_ids = fields.One2many(
+        comodel_name='account.analytic.line',
+        inverse_name='ticket_id',
+        string='Timesheet',
+    )
+    total_hours = fields.Float(
```

Comment:
This logic already exist on account.analytic.line

Reply:
Total hours is on the ticket not on the analytic.line...